src/main.py

```diff
@@ -126,14 +126,17 @@ def _run_pipeline(env: BaseEnv,
             approach.learn_from_offline_dataset(offline_dataset)
             learning_time = time.time() - learning_start
         # Run evaluation once before online learning starts.
-        results = _run_testing(env, approach)
-        results["num_transitions"] = total_num_transitions
-        results["query_cost"] = total_query_cost
-        results["learning_time"] = learning_time
-        _save_test_results(results, online_learning_cycle=None)
+        if CFG.skip_until_cycle < 0:
```

Comment:
is there a reason we need to indent this? the testing uses its own RNGs right?

Reply:
yeah, it's just because I want to skip the time to test when using this flag, because i'm impatient